lib/utils/helpers.js

```diff
@@ -26,6 +26,7 @@ const flattenMemberExpressionIfPossible = node => {
       if (node.computed || node.property.type !== 'Identifier') {
         return null
       }
+      // eslint-disable-next-line no-case-declarations
       const flattenedParent = flattenMemberExpressionIfPossible(node.object)
```

Comment:
this lint was failing lint check when I committed even though I didn't touch it, let me know what you want me to do